src/ArgSortMsg.chpl

```diff
@@ -338,7 +338,7 @@ module ArgSortMsg
         }
       }
       try! asLogger.debug(getModuleName(),getRoutineName(),getLineNumber(),
-                             "argsort time = %i".doFormat(Time.timeSinceEpoch().totalSeconds() - t1));
+                             "argsort time = %i".format(Time.timeSinceEpoch().totalSeconds() - t1));
```

Comment:
I wonder if it makes sense to pull this `doFormat` -> `format` change into it's own PR given that it's a huge number of changes dirtying this PR and it's only a single commit, so could easily be picked out. What do you think?

Reply:
Yeah, that's a good point. I can make a separate PR for it.

Reply:
PR here: https://github.com/Bears-R-Us/arkouda/pull/3365